src/test/java/ui/user/SignUpTest.java

```diff
@@ -0,0 +1,105 @@
+package ui.user;
+
+import org.openqa.selenium.WebElement;
+import org.realworld.ui.pages.HomePage;
+import org.realworld.ui.pages.SignUpPage;
+import org.realworld.ui.utils.SeleniumUtils;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import ui.base.UITestBase;
+
+import java.util.List;
+
+import static org.realworld.api.Constants.*;
+
+public class SignUpTest extends UITestBase {
+
+    @Test
+    public void signUpUserWithValidData() {
+        SignUpPage signUpPage = loadSignUpPage();
+        String userName = USER_NAME_PREFIX_UI + System.nanoTime();
+        HomePage homePage = new HomePage(signUpPage.signUp(userName, userName + EMAIL_SUFFIX, PASSWORD));
+        List<WebElement> menuItems = homePage.itemsLoggedInNavMenu();
+
+        SoftAssert sAssert = new SoftAssert();
+        sAssert.assertTrue(SeleniumUtils.getElementWithText(menuItems, userName) != null,
```

Comment:
Try to use `sAssert.executeAssert(SeleniumAssertion.elementExists("some xpath here"))`   
https://www.javadoc.io/static/org.testng/testng/6.8.8/org/testng/asserts/Assertion.html#executeAssert(org.testng.asserts.IAssert)

`elementExists` should return instance of `IAssert`

Reply:
Added isPresent method 
`public static Boolean isPresent(WebDriver driver, String xpath) {
        WebElement element = getElementWithWait(driver, xpath);
        return element != null;
    }`
So now it is more readable:
`sAssert.assertTrue(SeleniumUtils.isPresent(this.getDriver(), elementXpath),
                "Element by elementXpath:" + elementXpath + "is missing");`

Reply:
done